calibre-plugin/plugin-defaults.ini

```diff
@@ -1222,6 +1222,28 @@ extracategories:Lord of the Rings
 ## Site dedicated to these categories/characters/ships
 extracategories:Harry Potter
 
+[www.masseffect2.in]
+## Site dedicated to this fandom.
+extracategories: Mass Effect
+
+## Ucoz.com engine, upon which MassEffect2.in is based, imposes an unspecified limit on request frequency.
+## Sources vary from `5 requests per second' to `2 requests per second for more than 10 per minute'.
+## With default settings, a several-hours IP ban may follow, so set it lifted.
+slow_down_sleep_time: 2
+
+## Whether to exclude editor signature from the bottom if chapter text.
+exclude_editor_signature: false
+
+## Stories on the site almost never have cover image.
+## May be adjusted in `personal.ini' on per-story basis.
```

Comment:
I suggest removing "on per-story basis."  Because personal.ini settings cannot be per-story.